.github/workflows/trivy-pr-check.yaml

```diff
@@ -0,0 +1,18 @@
+name: 'Trivy Scan'
+on:
+  pull_request:
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run Trivy vulnerability scanner in repo mode
+        uses: aquasecurity/trivy-action@master
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-PIPELINE-0008   
**Check Name:** Unrestricted dependency version   
**Severity:** MEDIUM   
**Message:** Dependency aquasecurity/trivy-action master version should be pinned to the commit sha  
[This comment was created by Aqua Pipeline]